### PR TITLE
docs: improve self hosting analytics documentatoin

### PR DESCRIPTION
**Pull request by Ziinc**
Improve self-hosting analytics documentation, with slight re-organization to accommodate for multi backend support

cc @filipecabaco

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -325,7 +325,7 @@ services:
 
   analytics:
     container_name: supabase-analytics
-    image: supabase/logflare:1.3.28
+    image: supabase/logflare:1.4.1
```
Comment (filipecabaco) [inline docker/docker-compose.yml:328]: this will probably conflict as there was a change to move to 1.4.0 but minor

**Review by filipecabaco — APPROVED** [review]

File: apps/docs/docs/ref/self-hosting-analytics/introduction.mdx [hunk 2]
```diff
@@ -20,92 +20,100 @@ When self-hosting the Analytics server, the full logging experience matching tha
...
 
-- Postgres ingestion
-- Big Query ingestion
+When using the Postgres backend, tables are created for each source within the provided schema (for supabase-cli, this would be `_analytics`). Log events received by Logflare are inserted directly into the respective tables. All BigQuery-dialect SQL queries from Studio will be handled by a translation layer within the Analytics server. This translation layer translates the query to PostgreSQL dialect, and then executes it against the Postgres database.
 
-### Postgres Ingestion Requirements
+The Postgres backend is not yet optimized for high volume of inserts, or for heavy query usage. Furthermore, the translation layer only handles a limited subset of the BigQuery dialect such that the Studio is capable to work as per normal. As such, the [Log Explorer](https://supabase.com/docs/guides/platform/logs#logs-explorer) may have errors for more advanced queries when using the Postgres Backend.
```
Comment (gregnr) [inline apps/docs/docs/ref/self-hosting-analytics/introduction.mdx:36]:
```suggestion
The Postgres backend is not yet optimized for a high volume of inserts, or for heavy query usage. Today the translation layer only handles a limited subset of the BigQuery dialect. As such, the [Log Explorer](https://supabase.com/docs/guides/platform/logs#logs-explorer) may produce errors for more advanced queries when using the Postgres Backend.
```

File: apps/docs/docs/ref/self-hosting-analytics/introduction.mdx [hunk 3]
```diff
@@ -20,92 +20,100 @@ When self-hosting the Analytics server, the full logging experience matching tha
...
 
-- Requires you to use supabase/logflare:1.3.26 or above
-- Requires the following environment variables set:
-  - Required - POSTGRES_BACKEND_URL: `postgresql://{POSTGRES_USER}:{POSTGRES_PASSWORD}@{POSTGRES_HOST}:{POSTGRES_PORT}/{POSTGRES_DB}`
-  - Optional - POSTGRES_BACKEND_SCHEMA: `{SCHEMA_NAME}`
+## Getting Started
 
-### Big Query Ingestion Requirements
+The Postgres backend is recommended for getting familiar and experimenting with self-hosting Supabase. However, it is recommended to use the BigQuery backend for production scale. See [production recommendations](#production-recommendations) for more information
```
Comment (gregnr) [inline apps/docs/docs/ref/self-hosting-analytics/introduction.mdx:40]:
```suggestion
The Postgres backend is recommended when familiarizing and experimenting with self-hosting Supabase. For production, we recommend using the BigQuery backend. See [production recommendations](#production-recommendations) for more information.
```

File: apps/docs/docs/ref/self-hosting-analytics/introduction.mdx [hunk 4]
```diff
@@ -20,92 +20,100 @@ When self-hosting the Analytics server, the full logging experience matching tha
...
 
-- Google Cloud Account with billing enabled
-- Big Query enabled
+To get started with self-hosting Supabase, the [example docker-compose](https://github.com/supabase/supabase/tree/master/docker) will give you a jumpstart to setting up logging for the stack.
```
Comment (gregnr) [inline apps/docs/docs/ref/self-hosting-analytics/introduction.mdx:42]:
```suggestion
To set up logging in self-hosted Supabase, see the [docker-compose example](https://github.com/supabase/supabase/tree/master/docker).
```

File: apps/docs/docs/ref/self-hosting-analytics/introduction.mdx [hunk 5]
```diff
@@ -20,92 +20,100 @@ When self-hosting the Analytics server, the full logging experience matching tha
...
+Two compose services are required: Logflare, and Vector. Logflare is the HTTP Analytics server, while Vector is the logging pipeline to route all compose services' syslog to the Logflare sever.
 
-The remainder of the instructions are specified in the [Setup Big Query ingestion](#setup-big-query-ingestion) section.
+Regardless of backend chosen, the following environment variables **must** be set for the `supabase/logflare` docker image:
```
Comment (gregnr) [inline apps/docs/docs/ref/self-hosting-analytics/introduction.mdx:45]:
```suggestion
Regardless of the backend chosen, the following environment variables **must** be set for the `supabase/logflare` docker image:
```

File: apps/docs/docs/ref/self-hosting-analytics/introduction.mdx [hunk 6]
```diff
@@ -20,92 +20,100 @@ When self-hosting the Analytics server, the full logging experience matching tha
...
+  - `bigquery.routines.create`
+  - `bigquery.routines.update`
+  - `bigquery.tables.create`
+  - `bigquery.tables.delete`
+  - `bigquery.tables.get`
+  - `bigquery.tables.getData`
+  - `bigquery.tables.update`
+  - `bigquery.tables.updateData`
+
+You can create the service account via the web console or `gcloud`, as per the [Google Cloud documentation](https://cloud.google.com/iam/docs/keys-create-delete). In the web console, you can create the key by navigating to IAM > Service Accounts > Actions (dropdown) > Manage Keys
```
Comment (gregnr) [inline apps/docs/docs/ref/self-hosting-analytics/introduction.mdx:108]:
```suggestion
You can create the service account via the web console or `gcloud` CLI, as per the [Google Cloud documentation](https://cloud.google.com/iam/docs/keys-create-delete). In the web console, you can create the key by navigating to IAM > Service Accounts > Actions (dropdown) > Manage Keys
```

File: apps/docs/docs/ref/self-hosting-analytics/introduction.mdx [hunk 7]
```diff
@@ -194,6 +172,8 @@ The self-hosted example is only used as a minimal example on running the entire
 
 The current Postgres Ingestion backend isn't optimized for production usage. We recommend using Big Query for more heavy use cases.
 
+**We advise using the BigQuery backend for production settings, for better scaling and querying/debugging experience**.
```
Comment (gregnr) [inline apps/docs/docs/ref/self-hosting-analytics/introduction.mdx:175]:
```suggestion
**We recommend using the BigQuery backend for production environments as it offers better scaling and querying/debugging experiences.**
```

**Review by gregnr — APPROVED** [review]
Nice 👍

File: docker/docker-compose.yml (hunk 1 above)
Comment (Ziinc) [reply docker/docker-compose.yml]: reverted the change

**Pull request merged**